index.html

```diff
@@ -0,0 +1,122 @@
+                        <li><a href="index.html" class="mobile-a">Home</a></li>
+                        <li><a href="about.html" class="mobile-a">About</a></li>
+                        <li><a href="about.html" class="mobile-a">Join</a></li>
+                        <li><a href="#" class="mobile-a">Sponsor</a></li>
+                        <li><a href="#" class="button mobile-a">CC Campaign</a></li>
+                    </ul>
+        </nav>
+    </header>
+<main class="main-container">
+    <h1 class="first-h1">Hello! Welcome!!!</h1>
+        <h1 class="sec-h1">Meanwood Farmers Cooperative Conference</h1>
+            <p class="page-par">We take pride in helping out farmers get farming inputs on time in readiness for the farming season. Food security for our nation is our top priority
+            </p>
+            <p class="bold">
+           (MON)~(FRI) FROM 08:00am to 5:00pm
+            </p>
+            <p class="bottom-text">@ Meanwood Chamba Valley</p>
+</main>
+    <section class="main-section">
+        <h2 class="main-section-h1">
+            Main Program
+        </h2>
+            <div class="orange-underline"></div>
+                <ul class="blocks">
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Seminars</h2>
+                        <p>we offer seminars to farmers every two months
+                            per year. We attach importance to collabos
+                        </p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Lectures</h2>
+                        <p>Learn mordern farming techniques
+                            in our daily classes from monday to friday
+                            each week
+                        </p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Programs</h2>
+                        <p>Enrol and become eligible for our
+                            government sponsored susidies
+                        </p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Assistance</h2>
+                        <p>Access technical support
+                            form our technical teams
+                            available to you 24 hours a day
+                        </p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Acreditation</h2>
+                        <p>Become acredited with us and gain
+                            the advantages that come with being an
+                            affiliate.
+                        </p>
+                    </li>
+                </ul>
+                <button class="join-btn more-btn" type="button">Attend The Summit</button>
+            <a href="#" class="see-more">See more</a>    
+     </section>
```

Comment:
- [ ] The design of this section differs from the original design please fix it 🙏 including the right fonts, colors and margins

**Original:**
![original2](https://user-images.githubusercontent.com/38635158/124984654-2b121200-e042-11eb-8951-21db19f9bde9.png)
___
**Yours:**
![re](https://user-images.githubusercontent.com/38635158/124984668-2f3e2f80-e042-11eb-9513-adc15a469acc.png)